crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -223,8 +224,8 @@ pub struct Solutions {
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct Solution {
     id: u64,
-    #[serde_as(as = "HashMap<_, serialize::U256>")]
-    prices: HashMap<eth::H160, eth::U256>,
+    #[serde_as(as = "HashMap<_, _>")]
```

Comment:
I am no so sure we need this here.

Reply:
Since key and value now both use their regular deserialize implmentations this can be dropped AFAIK.